tests/framework/filters/stubs/UserIdentity.php

```diff
@@ -63,4 +64,19 @@ public function validateAuthKey($authKey)
     {
         throw new NotSupportedException();
     }
+
+    public function getRateLimit($request, $action)
```

Comment:
Doesn't look like a good idea to me. Interface should return an array in this case but it does not. Implementing methods like that isn't right. Better to leave default user implementation w/o this interface at all.